src/components/Frame/Frame.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import {MobileCancelMajorMonotone} from '@shopify/polaris-icons';
 import {classNames} from '@shopify/react-utilities/styles';
+import {durationBase} from '@shopify/polaris-tokens';
```

Comment:
Since you are using `duration(slow)` in the CSS this one should use `DurationSlow`.

The timeout on the `CSSTransition` needs to match the css timing.